BackendBench/backends/llm.py

```diff
@@ -370,6 +370,10 @@ def test_kernel_correctness(
 
             return is_correct, feedback_info
 
+        except AgentError as e:
+            feedback_info.compilation_error = str(e)
+            feedback_info.summary = f"Agent error: {str(e)}"
```

Comment:
What I meant here was add an agent_error field to `feedback_info`. 

We should not classify this as a compilation error. 
Also you should not modify the summary here.